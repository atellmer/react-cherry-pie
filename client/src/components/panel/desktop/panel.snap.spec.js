import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';

import TmPanelDesktop from './index';


const props = {
  dialogs: []
};

test('Jest: TmPanelDesktop (Snapshot)', () => {
  const shallowRenderer = ReactTestUtils.createRenderer();

  const result = shallowRenderer.render(<TmPanelDesktop {...props}/>);
  expect(result).toMatchSnapshot();
});
