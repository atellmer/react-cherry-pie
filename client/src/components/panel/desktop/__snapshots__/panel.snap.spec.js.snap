exports[`test Jest: TmPanelDesktop (Shallow Snapshot) 1`] = `
<div
  className={undefined}>
  <div
    className={undefined}>
    <Flex
      align="center"
      className={undefined}>
      <TmLogo />
    </Flex>
    <Flex
      align="center"
      className={undefined}>
      <TmSearchbar />
    </Flex>
  </div>
  <div>
    <TmTabs
      dialogs={Array []} />
  </div>
</div>
`;
